book/005-asynchronous-tooling.md

```diff
@@ -333,7 +333,7 @@ function getUsersTopComment(userInfo) {
 }
 ```
 
-By doing this we're able to keep single responsibility with our Promises, and have 2 functions we can use through composition and reuse, than creating giant functions we only need to use a few times.
+By doing this we're able to keep single responsibility with our Promises and have 2 functions we can use through composition and reuse, rather than creating giant functions we only need to use a few times.
```

Comment:
You caught something important here. This reads funny without the implied comma pause:

`By doing this, we're able to...` (continue rest of your awesome edits 👍 )